src/app.js

```diff
@@ -1,21 +1,56 @@
+/* eslint-disable no-console */
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
+const http = require('http');
+const fs = require('fs');
+const path = require('path');
+
+const PORT = 4200;
+const publicFolder = path.join(__dirname, 'public');
+
+function createServer() {
+  return http.createServer((req, res) => {
+    const filePath = getFilePath(req.url);
+
+    if (!filePath) {
+      res.writeHead(404);
+      res.end('File not found');
+
+      return;
+    }
+
+    fs.readFile(filePath, 'utf8', (err, data) => {
```

Comment:
Most files are not encoded in `utf-8`. 

```suggestion
    fs.readFile(filePath, (err, data) => {
```